app/src/main/java/com/realityexpander/tasky/agenda_feature/presentation/agenda_screen/AgendaScreen.kt

```diff
@@ -484,7 +495,81 @@ fun AgendaScreenContent(
                     }
                 }
 
-                if (index < agendaItems.size - 1) {
+                if (index < agendaItemsBeforeNow.size - 1) {
+                    Spacer(modifier = Modifier.smallHeight())
+                }
+            }
```

Comment:
Alternatively, you can use `verticalArrangement = Arrangement.spacedBy(X.dp)` which will already consider this (only works if you want to apply this spacing to the whole LazyColumn

Reply:
Since the Time Needle is not using spacing, I will keep the current implementation. Nice to know that the `spaceBy` works for lazylist, like a `spaceBetween`